### PR TITLE
Convention models facade

### PR DESCRIPTION
**Pull request by cristianfchq**
Code convention in folder models > facade

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/models/facade/ZipFileFacade.java [hunk 1]
```diff
@@ -55,18 +55,19 @@ public static void getZipFileVideo(final ParameterOutputChecksum parameterOutput
...
+     * @param storagePath is the path's name of file to compress.
      * @param thumbnail if add thumbnail of video into zip.
      * @throws IOException when invalid path is given in zipFiles.
      * @throws InterruptedException is exception if process is interrupted.
      */
-    private static void getZipFile(final ParameterOutputChecksum parameterOutputChecksum, final String metadata,
-                                   final String storagePath, final boolean thumbnail)
-            throws IOException, InterruptedException {
+    private static void getZipFile(final ParameterOutputChecksum parameterOutputChecksum,
+                                   final String metadata,
```
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/models/facade/ZipFileFacade.java]: There should not be one parameter per line, only if it passes 120 characters

File: src/main/java/org/fundacion/jala/converter/models/facade/ExtractFacade.java [hunk 2]
```diff
@@ -21,7 +21,7 @@
 import java.io.IOException;
 
 /**
- * Class to do and call facade of extract.
+ * This class calls facade of extract.
  */
 public class ExtractFacade {
     public static final String TXT_FILE_EXTENSION = ".txt";
```
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/models/facade/ExtractFacade.java:27]: You shouldn't use "static" variables

File: src/main/java/org/fundacion/jala/converter/models/facade/CompilerFacade.java [hunk 3]
```diff
@@ -7,7 +7,6 @@
  * license agreement you entered into with Fundacion Jala
  *
  * @author Jorge Caceres Velasco
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/facade/CompilerFacade.java]: Should be the full name of the author

File: src/main/java/org/fundacion/jala/converter/models/facade/ConverterFacade.java [hunk 4]
```diff
@@ -15,15 +15,19 @@
...
+/**
+ * This class calls facade of converter.
+ */
 public class ConverterFacade {
 
     private ConverterFacade() {
     }
 
     /**
      * Converts an audio file.
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/facade/ConverterFacade.java]: The description of this method seems to be incorrect. The name of the method is getAudioConverter

File: src/main/java/org/fundacion/jala/converter/models/facade/ConverterFacade.java [hunk 5]
```diff
@@ -33,8 +37,9 @@ public static String getAudioConverter(final AudioParameter audioParameter) {
 
     /**
      * Converts a video file.
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/facade/ConverterFacade.java]: The description of this method seems to be incorrect. The name of the method is getVideoConverter

File: src/main/java/org/fundacion/jala/converter/models/facade/CompilerFacade.java (hunk 3 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/models/facade/CompilerFacade.java]: ok Thanks for reminding me

File: src/main/java/org/fundacion/jala/converter/models/facade/ExtractFacade.java (hunk 2 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/models/facade/ExtractFacade.java:27]: Thanks for reminding me

File: src/main/java/org/fundacion/jala/converter/models/facade/ConverterFacade.java (hunk 5 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/models/facade/ConverterFacade.java]: ok. It was solved.

File: src/main/java/org/fundacion/jala/converter/models/facade/ConverterFacade.java (hunk 4 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/models/facade/ConverterFacade.java]: ok It was solved

File: src/main/java/org/fundacion/jala/converter/models/facade/ZipFileFacade.java (hunk 1 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/models/facade/ZipFileFacade.java]: ok. It was solved

File: src/main/java/org/fundacion/jala/converter/models/facade/ExtractFacade.java [hunk 6]
```diff
@@ -42,6 +43,7 @@ public static void getTextExtract(final ExtractTextParameter extractTextParamete
 
     /**
      * Extracts metadata from file.
+     *
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/facade/ExtractFacade.java:46]: Dots are missing at the end of the java docs in this class

Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/models/facade/ExtractFacade.java:46]: ok

**Review by danielaSantaCruz — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]

**Pull request merged**